urdf/stereolabs_zed.urdf.xacro

```diff
@@ -94,6 +130,12 @@ limitations under the License.
       <xacro:property name="screw_offset_z" value="-0.0155" />
       <xacro:property name="optical_offset_x" value="0.0" />
       <xacro:property name="mono" value="false" />
+      <xacro:property name="collision">
+        <origin xyz="${screw_offset_x} 0 0" rpy="0.0 0.0 0.0" />
+        <geometry>
+          <box size="0.05 0.25 0.25" />
```

Comment:
Are this values correct? I found following dimensions on internet: 31mm x 31mm x 52mm and for zedxone4k: 31mm x 31mm x 58.1mm